### DISCUSSION TITLE
Mouse issue when in emulation (not in AmiBerry GUI)

### DISCUSSION THREAD

**Discussion by Timothy-Neal** [BlitterStudio/amiberry#1009] [Q&A] [answered]
Hey Everyone,
hoping you can help :)

Issue:
When loading into an emulation the mouse cursor doesn't move. It just sits in the top left hand corner, But weirdly the mouse works in Amiberry perfectly and also if I right click (in the emulation)  I get options (see below) (so it's like it knows the mouse is there but for some reason doesn't know what to do with the movement). In Winuae you can configure this but can’t see a way to do it in amiberry. Remap in mouse settings is grayed out see attached
![Capture (Small)](https://user-images.githubusercontent.com/60937765/188356202-0
![20220902_110023 (Small)](https://user-images.githubusercontent.com/60937765/188356268-9cbdeec8-65a0-45f7-982a-9db5ef5a6fe1.jpg)
92549f9-3cec-42c9-9df1-0cde3aac2a96.PNG)
![20220902_110128 (Small)](https://user-images.githubusercontent.com/60937765/188356243-684bdecd-c076-4403-b831-0e0a18ac8b57.jpg)
![20220902_110115 (Small)](https://user-images.githubusercontent.com/60937765/188356273-631ede5a-6210-48b1-a12d-266170e0f193.jpg)

Background:
Using DietPi_RPi-ARMv6-Bullseye, Installed and updated 
Installed AmiBerry Via DietPi-Software
Tried different rom’s/kicks
Changed mouse from Wireless to wired and did a reinstall of dietpi and amiberry
Check all the settings i can and can’t see were its gone wrong
Read that sometimes sdl drivers that come with installs don’t work so installed one from the github instructions ( sudo apt install libsdl2-2.0-0 libsdl2-ttf-2.0-0 libsdl2-image-2.0-0 flac mpg123 libmpeg2-4)
Rebooted and still same issue

I am new to Raspberrypi and Linux in general, please be gentle :)

thanks in advance

**Comment by solskogen** [comment]
If you're new to RaspberryPi and Linux, I would not recommend going for DietPi. Go rather with the default Raspberry Pi OS.

**Reply by Timothy-Neal** [reply]
Thanks Solskogen, Ill continue to have a play and will probably do what you recommend :) thanks for your reply

**Reply by Timothy-Neal** [reply]
Installed on default pi os using PIkiss and works perfectly.
thanks again

**Comment by Timothy-Neal** [comment, marked as answer]
Installed on default pi os using PIkiss and works perfectly.
